src/main/java/eu/deltasource/audioplayer/player/PlayerFactory.java

```diff
@@ -0,0 +1,43 @@
+package eu.deltasource.audioplayer.player;
+
+import eu.deltasource.audioplayer.playable.audioplayable.Author;
+import eu.deltasource.audioplayer.playable.audioplayable.Song;
+import eu.deltasource.audioplayer.playlist.SongList;
+
+import static eu.deltasource.audioplayer.playable.audioplayable.AudioPlayableGenre.CLASSIC;
+import static eu.deltasource.audioplayer.playable.audioplayable.AudioPlayableGenre.POP;
+import static eu.deltasource.audioplayer.playable.audioplayable.AudioPlayableGenre.ROCK;
+
+public class PlayerFactory {
+
+    public static Player createPlayer(String playerType) {
+        if (playerType == null) {
```

Comment:
make sure you're handling the case when the player is null where you're using this method